examples/autorenderhint/src/index.js

```diff
@@ -0,0 +1,154 @@
+let startVideoBitrateGraph = null;
+let stopVideoBitrateGraph = null;
+
+/**
+ * Set up the bitrate graph for audio or video media.
+ */
+ function setupBitrateGraph(kind, containerId, canvasId) {
+  const bitrateSeries = new DataSeries();
+  const bitrateGraph = new GraphView(containerId, canvasId);
+
+  bitrateGraph.graphDiv_.style.display = 'none';
+  return async function startBitrateGraph(room, intervalMs) {
+    let bytesReceivedPrev = 0;
+    let timestampPrev = Date.now();
+    const interval = setInterval(async function() {
+      if (!room) {
+        clearInterval(interval);
+        return;
+      }
+      const stats = await room.getStats();
+      const remoteTrackStats = kind === 'audio'
+        ? stats[0].remoteAudioTrackStats[0]
+        : stats[0].remoteVideoTrackStats[0]
+      const bytesReceived = remoteTrackStats.bytesReceived;
+      const timestamp = remoteTrackStats.timestamp;
+      const bitrate = Math.round((bytesReceivedPrev - bytesReceived) * 8 / (timestampPrev - timestamp));
+
+      bitrateSeries.addPoint(timestamp, bitrate);
+      bitrateGraph.setDataSeries([bitrateSeries]);
+      bitrateGraph.updateEndDate();
+      bytesReceivedPrev = bytesReceived;
+      timestampPrev = timestamp;
+    }, intervalMs);
+
+    bitrateGraph.graphDiv_.style.display = '';
+    return function stop() {
+      clearInterval(interval);
+      bitrateGraph.graphDiv_.style.display = 'none';
+    };
+  };
+}
+
+const handleIsSwitchedOff = (trackState) => {
+  if(trackState) {
+    trackIsSwitchedOff.textContent = 'Off';
+    trackIsSwitchedOff.classList.remove('badge-success');
+    trackIsSwitchedOff.classList.add('badge-danger');
+  } else {
+    trackIsSwitchedOff.textContent = 'On';
+    trackIsSwitchedOff.classList.remove('badge-danger');
+    trackIsSwitchedOff.classList.add('badge-success');
+  }
+}
+
+(async function(){
+  // Load the code snippet.
+  const snippet = await getSnippet('./helpers.js');
+  const pre = document.querySelector('pre.language-javascript');
+
+  pre.innerHTML = Prism.highlight(snippet, Prism.languages.javascript);
+
+  const logger = Video.Logger.getLogger('twilio-video');
+  logger.setLevel('silent');
+
+  // Get the credentials to connect to the Room.
+  const credsP1 = await getRoomCredentials();
+  const credsP2 = await getRoomCredentials();
+
+  // Create room instance and name for participants to join.
+  roomP1 = await joinRoom(credsP1.token);
+
+  // Create the video track for the Remote Participant.
+  const videoTrack = await Video.createLocalVideoTrack();
+
+  // Connecting remote participant.
+  const roomP2 = await Video.connect(credsP2.token, {
+    name: 'my-cool-room',
+    bandwidthProfile: {
+      video: {
+        contentPreferencesMode: 'auto',
+        clientTrackSwitchOffControl: 'auto'
+      }
+    },
+    tracks: [ videoTrack ]
+  });
+
+  // Set video bitrate graph.
+  startVideoBitrateGraph = setupBitrateGraph('video', 'videobitrategraph', 'videobitratecanvas');
```

Comment:
is `startVideoBitrateGraph` referenced outside the function? if not see if you can get rid of this global.